build/pxc-entrypoint.sh

```diff
@@ -440,6 +440,8 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		wsrep_verbose_logfile=$(mktemp $DATADIR/wsrep_recovery_verbose.XXXXXX)
```

Comment:
looks like we need to remove all `/var/lib//wsrep_recovery_verbose.*` during start because the liveness probe fails if any exists.
in such case idea is to have always static filename `/var/lib//wsrep_recovery_verbose.log`.
if we want to store all recovery logs for history we can do `cat /var/lib//wsrep_recovery_verbose.log >> /var/lib//wsrep_recovery_verbose-history.log` before `rm` command